src/connectors/numeric-refinement-list/connectNumericRefinementList.js

```diff
@@ -213,42 +214,41 @@ export default function connectNumericRefinementList(renderFn, unmountFn) {
       },
 
       getWidgetSearchParameters(searchParameters, { uiState }) {
-        const clearedParams = searchParameters.clearRefinements(attributeName);
+        let clearedParams = searchParameters.clearRefinements(attributeName);
         const value =
           uiState.numericRefinementList &&
           uiState.numericRefinementList[attributeName];
-        if (value) {
-          const valueAsEqual = value.indexOf(':') === -1 && value;
-
-          if (valueAsEqual) {
-            return clearedParams.addNumericRefinement(
-              attributeName,
-              '=',
-              valueAsEqual
-            );
-          }
-
-          const [lowerBound, upperBound] = value.split(':');
-          if (
-            (lowerBound || lowerBound === 0) &&
-            (upperBound || upperBound === 0)
-          ) {
-            return clearedParams
-              .addNumericRefinement(attributeName, '>=', lowerBound)
-              .addNumericRefinement(attributeName, '<=', upperBound);
-          }
-          if (lowerBound || lowerBound === 0)
-            return clearedParams.addNumericRefinement(
-              attributeName,
-              '>=',
-              lowerBound
-            );
-          if (upperBound || upperBound === 0)
-            return clearedParams.addNumericRefinement(
-              attributeName,
-              '<=',
-              upperBound
-            );
+
+        if (!value) {
+          return clearedParams;
+        }
+
+        const valueAsEqual = value.indexOf(':') === -1 && value;
+
+        if (valueAsEqual) {
+          return clearedParams.addNumericRefinement(
+            attributeName,
+            '=',
+            valueAsEqual
+          );
+        }
```

Comment:
We didn't test that case, we only test when it's a range.

Reply:
Test were added